src/components/Metadata/Metadata.tsx

```diff
@@ -39,69 +39,74 @@ const propsCompare = (_, newProps: MetadataProps) => {
 
 // TODO: Refactor loading state logic
 // TODO: Split "Metadata" component into "Metadata" and "PrivateMetadata" components
-export const Metadata: React.FC<MetadataProps> = memo(
-  ({ data, onChange, isLoading, readonly = false, hidePrivateMetadata = false, ...props }) => {
-    const change = (event: ChangeEvent, isPrivate: boolean) => {
-      const { action, field, fieldIndex, value } = parseEventData(event);
-      const key = getDataKey(isPrivate);
-      const dataToUpdate = data[key];
+export const MetadataNoMemo: React.FC<MetadataProps> = ({
```

Comment:
question: Is it any sens to have exported two component memenoize and not?

Reply:
Yeah... it's not ideal: I needed it to use for new form because previous component used memoization for preventing it from having loading state. I didn't want to break any existing usage though, since it's used all over the place